pkg/license/identifier.go

```diff
@@ -19,58 +19,48 @@ package license
 
 import (
 	"fmt"
-	"io/fs"
 	"path/filepath"
 	"regexp"
-	"strings"
 	"sync"
 
+	"github.com/google/licensecheck"
+
 	"github.com/apache/skywalking-eyes/assets"
 	"github.com/apache/skywalking-eyes/internal/logger"
 )
 
-var licenseTemplatesDir = "lcs-templates"
+const (
+	// coverageThreshold is the minimum percentage of the file that must contain license text.
+	// Reference: https://github.com/golang/pkgsite/blob/d43359e3a135fc391960db4f5800eb081d658412/internal/licenses/licenses.go#L48
+	coverageThreshold = 75
 
-var templatesDirs = []string{
-	licenseTemplatesDir,
-	// Some projects simply use the header text as their LICENSE content...
-	"header-templates",
-}
+	licenseTemplatesDir = "lcs-templates"
+)
 
-var dualLicensePatterns = []*regexp.Regexp{
-	regexp.MustCompile(`(?i)This project is covered by two different licenses: (?P<license>[^.]+)`),
-}
+var (
+	_scanner    *licensecheck.Scanner
+	scannerOnce sync.Once
 
-var normalizedTemplates = sync.Map{}
+	dualLicensePatterns = []*regexp.Regexp{
+		regexp.MustCompile(`(?i)This project is covered by two different licenses: (?P<license>[^.]+)`),
+	}
+)
```

Comment:
Hey @fgksgf , I found `coverage.Match` is exactly what we want here, `coverage.Match` is an array of licenses if it's dual-license, just like what you put in the test, so we can simply remove this pattern and return the `coverage.Match` (for example, joined by `AND`)